Flick/Controllers/ListViewController.swift

```diff
@@ -94,12 +125,32 @@ class ListViewController: UIViewController {
             make.top.equalTo(listNameLabel.snp.bottom).offset(20)
             make.leading.trailing.bottom.equalToSuperview()
         }
+
+        // Empty list
+        if list.shows.count == 0 {
+            addMediaMessageLabel.snp.makeConstraints { make in
+                make.top.equalTo(arrowToAddButtonView.snp.bottom)
+                make.trailing.equalTo(arrowToAddButtonView.snp.leading)
+            }
+
+            arrowToAddButtonView.snp.makeConstraints { make in
+                make.top.equalToSuperview().offset(260)
+                make.trailing.equalToSuperview().inset(40)
+                make.size.equalTo(arrowSize)
+            }
+
+            emptyListImageView.snp.makeConstraints { make in
+                make.centerX.equalToSuperview()
+                make.top.equalToSuperview().offset(280)
+                make.size.equalTo(emptyListSize)
+            }
+        }
```

Comment:
Again, I think we may be able to move this entirely to its own setupEmptyView() function